py/tests/test_pipeline.py

```diff
@@ -0,0 +1,65 @@
+import os
+from typing import Any
+
+from pyflink.datastream import DataStream, DataStreamSink, StreamExecutionEnvironment
+from sentry_streams.adapters.flink_adapter import FlinkAdapter
+from sentry_streams.adapters.stream_adapter import StreamAdapter
+from sentry_streams.pipeline import Pipeline, RuntimeTranslator
+from sentry_streams.runner import iterate_edges
+
+
+# Essentially identical to runner for now
+# Checks basic input and output stream types
+# of a simple Flink program that uses
+# Pipeline abstractions
+def test_pipeline() -> None:
```

Comment:
A first test, kind of an integration test. It feels to me that this is the most useful test for the current state of the repo. Most of the functionality in the repo so far is to support a simple e2e pipeline like the one this test has.